apps/docs/content/docs/usage.mdx

```diff
@@ -6,15 +6,13 @@ date: 3/22/2023
 order: 2
 ---
 
-The `<Tldraw/>` component is the easiest way to get started with tldraw. It combines several lower-level libraries: the `TldrawEditor` component from `@tldraw/editor` and the `TldrawUi` component from `@tldraw/ui`.
-
-To use the `<Tldraw/>` component, create a file like this one:
+The `<Tldraw>` component is the easiest way to get started. To use it, create a file like this one:
 
 ```tsx
 import { Tldraw } from '@tldraw/tldraw'
 import '@tldraw/tldraw/tldraw.css'
 
-export default function TldrawExample() {
+export default function () {
```

Comment:
Changed this so it's the same as the one on the Introduction page. If it's different, people might think that it's different for a specific reason. If we want to keep the function name, we should add one to the Introduction page too, so that it's consistent.